lib/tasks/update_data.rake

```diff
@@ -0,0 +1,25 @@
+task update_data: :environment do
+  items = Item.all
+  puts "Going to update #{items.count} items"
+
+  ActiveRecord::Base.transaction do
+    items.each do |item|
```

Comment:
Imagine you had 1 million items in your database. This would load 1 million ActiveRecord objects in to memory and iterate through them, which probably would consume all of your memory. What you really want is something like [#find_each](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html). Get rid of the call to .all and just use `Item.count` to get the count, then use #find_each to iterate through all the records.